vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -80,7 +83,7 @@
  * @see <a href="https://developer.vimeo.com/api">The Vimeo API Docs</a>
  */
 @SuppressWarnings("unused")
-public class VimeoClient {
+final public class VimeoClient {
```

Comment:
My understanding is that the reason for the custom callback was to provide a hook for the old bad custom deserialization as well as error conversion and providing a simpler API for consumption (`success(T)` over `onResponse(Call<T>, Response<T>)`) and enforcing this pattern. The problem is that it does not provide a way to cancel and so really exposing the `Call<T>` is still necessary which makes for this ambiguous API.

That said, I think the above reasons mentioned are unnecessary and could be better served by just exposing the call object.